composables/useMigrate.ts

```diff
@@ -123,13 +123,15 @@ export function useMigrateDeposit(
     return 12
   })
 
-  const chainItemDeposit = computed(() =>
-    parseDeposit(
+  const chainItemDeposit = computed(() => {
+    // Calculate the sum of all deposits and then multiply by itemCount squared
```

Comment:
Why?

Reply:
Previously, I tried to migrate some of the items, on the deposit section shows my balance is enough. But, on the extrinsic, it failed https://assethub-kusama.subscan.io/extrinsic/0xf2759f46c9a5ac6e3e9b7f0823f7b2e245651941b32abefeab30b8b0220ecf4b